app/Services/BulkService.php

```diff
@@ -405,7 +405,11 @@ protected function updateRelations(array $filledFields)
             if ($relation->owner_id == Arr::get($filledFields, 'target_id') || ($relation->target_id == Arr::get($filledFields, 'owner_id'))) {
                 continue;
             }
-
+            if (request()->update_mirrored && $relation->mirror) {
+                $relation->mirror->update($filledFields);
+                $this->count++;
+                $this->total++;
```

Comment:
This will be a problem if you bulk update the target and push the changes to the mirror (as the mirror is now attached to itself from itself, causing a buggy loop).
The $filledFields should exclude `target_id` and `owner_id`, to avoid relations from one entity to itself :)

The count and total should only increment if there was something to update (`update()` returns true)